src/base/os.lua

```diff
@@ -11,7 +11,6 @@
 --
 
 	premake.override(os, "execute", function(base, cmd)
-		cmd = path.normalize(cmd)

```

Comment:
Calling path.normalize on os.execute() does seem dodgy. It will treat all the arguments of the command as one long path and munge the arguments.

For example: `os.execute("echo a/b ../c")` will output `a/c`

I don't think we want path.normalize doing special work to figure out if you passed in one path, or multiple separate paths, that would almost always be surprising for the caller, so the two neat solutions I can image are:

1. os.execute() takes multiple arguments. `os.execute("echo","a/b","../c")`. Not quite sure how that could map to os.executef().
2. Teach os.execute() to parse command lines.

Or, 3, stop os.execute() being clever and normalizing paths (what does the normalization functionally add?).

Thoughts?